policies/rudderc/tests/cases/general/ntp/technique.ps1

```diff
@@ -35,7 +35,7 @@
         TechniqueName = $techniqueName
     }
     
-    $class = ""debian""
+    $class = ""false""
```

Comment:
these double quotes look dubious to me

Reply:
Double double quotes does not exist in powershell